frontend/app-preview/src/views/LandingPage.tsx

```diff
@@ -4,79 +4,112 @@ import { PreviewContext } from '../PreviewContext';
+import { Button, ToggleButtonGroup } from '@digdir/design-system-react';
+import { ButtonVariant } from '@altinn/altinn-design-system';
+import { LinkIcon, ArrowCirclepathIcon, EyeIcon } from '@navikt/aksel-icons';
+import { AltinnHeader } from 'app-shared/components/altinnHeader';
+import { buttonActions, subMenuContent } from 'app-development/layout/PageHeader';
+import { AltinnHeaderVariant } from 'app-shared/components/altinnHeader/types';
+import { IRepository } from 'app-shared/types/global';
 
-export const LandingPage = () => {
+export interface LandingPageProps {
+  showSubMenu: boolean;
+  variant?: AltinnHeaderVariant;
+  repository?: IRepository;
+  subMenuContent?: JSX.Element;
+}
+
+export const LandingPage = ({ showSubMenu, variant = 'preview', repository }: LandingPageProps) => {
   const { org, app } = useParams();
   const { t } = useTranslation();
   const previewConnection = usePreviewConnection();
-  const { data: instanceId } = useInstanceIdQuery(org, app);
-  const selectedLayoutInEditor = localStorage.getItem(instanceId);
   const localSelectedViewSize = localStorage.getItem('viewSize');
   const [viewSize, setViewSize] = useState<string>(localSelectedViewSize ?? 'desktop');
   const selectedLayoutSetInEditor = localStorage.getItem('layoutSetName');
-
-
+  const { data: user } = useUserQuery();
   const isIFrame = (input: HTMLElement | null): input is HTMLIFrameElement =>
     input !== null && input.tagName === 'IFRAME';
 
   if (previewConnection) {
-    previewConnection.on("ReceiveMessage", function (message) {
+    previewConnection.on('ReceiveMessage', function (message) {
       const frame = document.getElementById('app-frontend-react-iframe');
       if (isIFrame(frame) && frame.contentWindow) {
         const targetOrigin = window.origin;
         // Trigger a reload of preview window until app-frontend implements re-calling api #https://github.com/Altinn/app-frontend-react/issues/1088
         window.location.reload();
-        console.log("Sending reload message to app-frontend with targetOrigin: " + targetOrigin);
+        console.log('Sending reload message to app-frontend with targetOrigin: ' + targetOrigin);
         frame.contentWindow.postMessage({ action: message }, targetOrigin);
       }
-    })
+    });
   }
 
   const handleChangeViewSizeClick = (selectedViewSize: string) => {
-      localStorage.setItem('viewSize', selectedViewSize);
-      setViewSize(selectedViewSize);
+    localStorage.setItem('viewSize', selectedViewSize);
+    setViewSize(selectedViewSize);
   };
 
   return (
-      <PreviewContext>
+    <PreviewContext>
+      <>
         <div className={classes.header}>
-          <a href={`/editor/${org}/${app}/ui-editor?layout=${selectedLayoutInEditor}`}>
-            <AltinnStudioLogo />
-          </a>
-          <div className={classes.betaTag}>
-            {'BETA'}
-          </div>
+          <AltinnHeader
+            menu={undefined}
+            showSubMenu={showSubMenu}
```

Comment:
Since we want to show the sub-menu for preview, this could just be set to `true` explicitly.
```suggestion
            showSubMenu={true}
```

Reply:
done